casts/splitNavigator/src/SplitNavigator/SplitBottomTabBar.tsx

```diff
@@ -0,0 +1,305 @@
+/* eslint-disable react/no-unused-prop-types */
+import * as React from 'react';
+import { View, ImageSourcePropType, StyleProp, ViewStyle, StyleSheet } from 'react-native';
+import {
+    GestureEvent,
+    NativeViewGestureHandlerPayload,
+    NativeViewGestureHandlerProps,
+    RawButton as GHRawButton,
+    RawButtonProps,
+} from 'react-native-gesture-handler';
+import ReAnimated, {
```

Comment:
It's the first time you've decided to name it as `ReAnimated`, not `Animated`)

Reply:
Yeah, I had a conflict between names, but then I removed the one from RN, but thought that ReAnimated sounds kinda cool :D